makeTask.js

```diff
@@ -0,0 +1,24 @@
+const Mustache = require('mustache');
+const fs = require('fs');
+const _ = require('lodash');
+require('dotenv').config({silent: true});
+
+var options = {
+    encoding: 'utf8'
+}
+
+var template = fs.readFileSync('./ecs-task-template.mustache', options);
+
+var view = {};
+
+view.env = _.chain(process.env)
+.pick(process.env, ['SLACK_API_TOKEN', 'SLACK_API_TOKEN_AWS', 'GITHUB_API_TOKEN', 'GITHUB_USERNAME', 'VERSION'])
```

Comment:
You don't want `SLACK_API_TOKEN_AWS` in here, because it's never used in your code.